otter/test/convergence/test_gathering.py

```diff
@@ -702,3 +706,89 @@ def test_no_group_servers(self):
             get_rcv3_contents=_constant_as_eff((), []))
 
         self.assertEqual(resolve_stubs(eff), {'servers': [], 'lb_nodes': []})
+
+
+class GetAllStacksTests(SynchronousTestCase):
+    """Tests for :func:`get_all_stacks`."""
+
+    def test_default(self):
+        """
+        Tests for passing no arguments.
```

Comment:
In general we try to describe what should happen in a test docstring instead of what feature is being tested. Here's a nice short guideline on the subject: https://jml.io/pages/test-docstrings.html

OTOH, don't be afraid to keep them short. It's also okay to keep the triple-quotes on the same line as the content as long as it's short enough to fit on one line.

Here I would say something like "Passing no arguments causes all stacks to be requested."